### PR TITLE
PEPPER-665 . participant list down Race's ethnicity (nested options) fix

### PR DESCRIPTION
**Pull request by ssettipalli**
PEPPER-665

Include RACE ethnicity which are selected nested options of the Picklist question in download

### CODE REVIEW

**Comment by ssettipalli** [comment]
[DataDictionary.xlsx](https://github.com/user-attachments/files/16640234/DataDictionary.xlsx)
[Participant-2024-08-16.xlsx](https://github.com/user-attachments/files/16640235/Participant-2024-08-16.xlsx)
[DataDictionary-2.xlsx](https://github.com/user-attachments/files/16640240/DataDictionary-2.xlsx)
[Participant-2024-08-16-2.xlsx](https://github.com/user-attachments/files/16640241/Participant-2024-08-16-2.xlsx)

**Review by denniscunningham — CHANGES_REQUESTED** [review]
There is test infrastructure in place for this code already, so it would be great to see some tests that verify this new behavior.

File: pepper-apis/dsm-core/src/test/resources/elastic/lmsAboutYouActivity.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+[
```
Comment (ssettipalli) [inline pepper-apis/dsm-core/src/test/resources/elastic/lmsAboutYouActivity.json:1]: activity instance / survey response for testing

File: pepper-apis/dsm-core/src/test/resources/elastic/lmsAboutYouActivityDef.json [hunk 2]
```diff
@@ -0,0 +1,460 @@
+{
```
Comment (ssettipalli) [inline pepper-apis/dsm-core/src/test/resources/elastic/lmsAboutYouActivityDef.json:1]: activity definition

File: pepper-apis/dsm-core/src/test/resources/elastic/participantListAboutYouRaceDownload.json [hunk 3]
```diff
@@ -0,0 +1,515 @@
+{
```
Comment (ssettipalli) [inline pepper-apis/dsm-core/src/test/resources/elastic/participantListAboutYouRaceDownload.json:1]: filters

**Comment by ssettipalli** [comment]
> There is test infrastructure in place for this code already, so it would be great to see some tests that verify this new behavior.

Added test case to DownloadParticipantListServiceTest which checks for nested options behavior in elastic and download

**Review by denniscunningham — APPROVED** [review]

**Pull request merged**